products/eks.md

```diff
@@ -102,19 +114,23 @@ releases:
 > conformant for a predictable experience.
 
 Amazon EKS guarantees support for at least four production-ready versions of Kubernetes at any given
-time. A Kubernetes version is fully supported on EKS for 14 months after first being available on
-Amazon EKS. This is true even if upstream Kubernetes is no longer supporting a version available on
-Amazon EKS.
+time. A Kubernetes version is fully supported on EKS for 14 months as standard support and for up to
+26 months as exteneded support after first being available on Amazon EKS. This is true even if
+upstream Kubernetes is no longer supporting a version available on Amazon EKS.
 
 You can subscribe to upgrade notices (sent approximately 12 months after the Kubernetes version was
 released on Amazon EKS) on your [Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/).
 The notice includes the end of support date, which is at least 60 days from the date of the notice.
 
 ## Upgrading
 
+At the end of the of standard support for a Kubernetes version, Amazon EKS clusters running on EOL
```

Comment:
It's only a preview for now according to https://aws.amazon.com/jp/blogs/containers/amazon-eks-extended-support-for-kubernetes-versions-available-in-preview/.

I think this should also be documented with a link to this article, notably:

- What does "preview" mean ?
- When did it start ?
- For which minimal Kubernetes version ?

Reply:
Kubernetes v1.23 is the oldest version that is currently supported by AWS and the oldest version that has received extended support.

I think we can assume that all Kubernetes clusters below v1.23 have already been automatically upgraded to v1.23, so noting the minimum Kubernetes version is only of historical interest when the process has changed.

Is endoflife.date also supposed to document the historical process (not applicable to any clusters anymore), or only the current process?

Reply:
> Is endoflife.date also supposed to document the historical process (not applicable to any clusters anymore), or only the current process?

Mostly the current process, but during a period of transition I think it's best to explain a little bit more. And same as above, considering this will be a paid option I would find odd that you can't opt-out of it.